ddpui/ddpdbt/dbt_service.py

```diff
@@ -220,6 +220,30 @@ def check_repo_exists(gitrepo_url: str, gitrepo_access_token: str | None) -> boo
     return response.status_code == 200
 
 
+def make_edr_report_s3_path(org: Org):
+    """make s3 path for elementary report"""
+    todays_date = datetime.today().strftime("%Y-%m-%d")
+    bucket_file_path = f"reports/{org.slug}.{todays_date}.html"
+    return bucket_file_path
+
+
+def refresh_elementary_report(org: Org):
+    """refreshes the elementary report for the current date"""
+    running_task = TaskProgress.get_running_tasks(
+        f"{TaskProgressHashPrefix.RUNELEMENTARY}-{org.slug}"
+    )
+    logger.info("running_task %s", str(running_task))
+    if running_task and len(running_task) > 0:
+        logger.info("edr already running for org %s", org.slug)
+        task_id = running_task[0].decode("utf8")
+        return task_id
```

Comment:
```suggestion
        return None, {"task_id": task_id}
```
the return expects and error and response

Reply:
you're quite right